python/pyarrow/table.pxi

```diff
@@ -4599,7 +4599,7 @@ cdef class Table(_Tabular):
         """
         return self.drop_columns(columns)
 
-    def group_by(self, keys):
+    def group_by(self, keys, use_threads=True):
         """Declare a grouping over the columns of the table.
```

Comment:
You could try this, I've actually seen this fix "unused parameters" errors in numpydocs before:
```
"""
Declare a grouping over the columns of the table.
...
"""
```

Reply:
Oh .. and that indeed did it :man_shrugging: 
Thanks a lot!